packages/compat/src/compat-adapters/ember-cli-fastboot.ts

```diff
@@ -141,6 +141,11 @@ class RewriteManifest extends Plugin {
       extraVendorFiles,
     };
 
+    json['ember-addon'] = {
```

Comment:
i don't remember if this was actually important now, i think it isn't and should likely be removed